lib/react.dart

```diff
@@ -5,8 +5,12 @@
 /// A Dart library for building UI using ReactJS.
 library react;
 
+import 'package:react/react_client/react_interop.dart';
```

Comment:
react_client pieces must not be imported in this entrypoint, since it pollutes this with `dart:html`.